views/component/Page.njk

```diff
@@ -55,32 +55,32 @@
     serviceName: "Civil Service Learning",
     serviceUrl: "#",
     navigation: [
-      {
-        href: "#1",
-        text: "Home",
-        active: true
-      },
-      {
-        href: "#2",
-        text: "Suggestions for you"
-      },
-      {
-        href: "#3",
-        text: "Learning record"
-      },
-      {
-        href: "#4",
-        text: "Profile"
-      },
-      {
-        href: "#5",
-        text: "Admin"
-      },
-      {
-        href: "#6",
-        text: "Sign out"
-      }
-    ]
+               {
```

Comment:
STYLING - Either 2 spaces or 1 tab for everyone. Keep it consistent please!

Reply:
Cool will keep an eye on this going forward. I've reformatted this to look a little cleaner